vagrant/boxes.d/00-centos.yaml

```diff
@@ -30,3 +30,7 @@ boxes:
     scenarios:
       - foreman
       - katello
+
+  centos8:
```

Comment:
Odd indenting?

Reply:
what's odd here?

Reply:
Looks like Github renders it incorrectly in diff mode and that was incorrect.